source/simulator/assembly.cc

```diff
@@ -804,6 +913,14 @@ namespace aspect
                                                parameters.composition_degree)
                                               +
                                               (parameters.stokes_velocity_degree+1)/2),
+                                  //TODO: give this a 1-point QGauss to save space, as this is never going to be used here? (discontinuous temperature field doesn't require stabilization)
+                                  QGauss<dim-1>((advection_field.is_temperature()
```

Comment:
Thinking about this some more, you can create an invalid quadrature formula using `Quadrature<dim-1>()`. I think this is exactly what you want.